_shared/project/.github/workflows/ci.yml

```diff
@@ -4,24 +4,40 @@ on:
     paths-ignore:
       - '.cookiecutter/*'
       - '.github/dependabot.yml'
+{% if include_exists(".github/workflows/environments.json") %}
       - '.github/workflows/deploy.yml'
       - '.github/workflows/redeploy.yml'
+{% endif %}
+{% if cookiecutter._directory in ['pyapp', 'pyramid-app'] %}
```

Comment:
I assume there isn't a public API that provides the same info as `_directory`?

Reply:
The leading `_` here doesn't actually signify "private" as it usually does in Python. It's a cookiecutter thing: when you run the `cookiecutter` command to generate a new project it walks you interactively through choosing the values for all the cookiecutter's settings (What versions of Python do you want to support? Do you want to have Docker Compose services? Etc). Settings whose names begin with a leading underscore are settings that this interactive UI will skip (just using the default value), but projects can still set them either with command line arguments when running `cookiecutter` to create the project or by modifying the project's `cookiecutter.json` file and running `make template`